content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -201,10 +201,16 @@ for examples of using this capability.
 ## PROXY Protocol
 
 {{< boilerplate experimental >}}
+The [PROXY protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) allows for exchanging and preservation of client attributes between TCP proxies,
+without relying on L7 protocols such as HTTP and the `X-Forwarded-For` and `X-Envoy-External-Address` headers, and is intended for scenarios where an external TCP load balancer needs to proxy TCP traffic through an Istio gateway to a backend TCP service and still expose client attributes such as source IP to upstream TCP service endpoints. 
 
-The [PROXY protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) allows for exchanging and preservation of client attributes across multiple proxies without relying on Layer 7 protocols.
+{{< idea >}}
```

Comment:
This seems more like a warning than an idea?

```suggestion
{{< warning >}}
```

Reply:
It is, applied, thx!